lib/puppet/module.rb

```diff
@@ -58,7 +58,7 @@ def self.is_module_namespaced_name?(name)
   attr_accessor :dependencies, :forge_name
   attr_accessor :source, :author, :version, :license, :summary, :description, :project_page
 
-  def initialize(name, path, environment)
+  def initialize(name, path, environment, include_prerelease = true)
```

Comment:
No, it shouldn't. That's one of the places that I identified as a "root", i.e. where we need to make a decision.